### ISSUE TITLE
Repo Cleanup

### ISSUE THREAD

**Issue by HTRamsey** [mavlink/qgroundcontrol#10518]
General Discussion

**Comment by holden-zenith** [comment]
@booo what is the state of iOS integration? Is it still functional?

**Comment by booo** [comment]
> @booo what is the state of iOS integration? Is it still functional?

I have no expertise in this area.

> General Discussion

I guess this needs a bit more of explanation :)

**Comment by Davidsastresas** [comment]
What do we mean here for repo clean up? that is very broad!

Regarding IOS, what is the current status? Is there something not building or not working for IOS or we just don't have the appstore thing working?

If it is the second, If I recall correctly, it needs to be done manually, I think we can not do that in the CI. At least it was like that last time I looked into it, a couple years ago.

**Comment by booo** [comment]
@Davidsastresas or @fredowski can you summarize which commits need merging for the video support?

**Comment by Davidsastresas** [comment]
I started to get familiarized with it very recently, fredowsky have definitely more criteria than me to answer here.
